src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/is-empty.pipe.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { IsEmptyPipe } from './is-empty.pipe';
+
+describe('IsEmptyPipe', () => {
+  const pipe = new IsEmptyPipe();
+
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('should transform to true (1)', () => {
+    expect(pipe.transform(NaN)).toBeTruthy();
+  });
+
+  it('should transform to true (2)', () => {
+    expect(pipe.transform(undefined)).toBeTruthy();
+  });
```

Comment:
What's the value of testing that the external `isEmpty()` function works as expected? [`lodash` already has more comprehensive unit tests](https://github.com/lodash/lodash/blob/master/test/isEmpty.js) for that purpose?

However, this is not testing the key thing this change is bringing in (reducing change detection events for functions in templates), which, for example, [`ngx-function-pipe` actually does](https://github.com/ArtemLanovyy/ngx-pipe-function/blob/master/__tests__/pipe-function.pipe.spec.ts#L73).

That's my point with using external libraries: you not only get 'working code' for free, you may also get higher quality (via more comprehensive unit tests), and a community supporting that and [reporting issues](https://github.com/ArtemLanovyy/ngx-pipe-function/issues).